dali/test/python/test_RN50_external_source_parallel_utils.py

```diff
@@ -112,11 +110,13 @@ def __init__(self, data_path, batch_size):
         self.batch_size = batch_size
 
     def __call__(self, batch_info):
-        assert isinstance(batch_info, types.BatchInfo), \
-            "Expected batch info instance, got {}".format(type(batch_info))
+        assert isinstance(batch_info,
+                          types.BatchInfo), f"Expected batch info instance, got {type(batch_info)}"
```

Comment:
That seems... random. The original formatting was more logical.
```suggestion
        assert isinstance(batch_info, types.BatchInfo), \
               f"Expected batch info instance, got {type(batch_info)}"
```